### ISSUE TITLE
execution-status-count does not include the "starting" case

### ISSUE THREAD

**Issue by SHuang-Broad** [broadinstitute/cromshell#126]
I'm running into a possible edge case where the gather step in a 3000 scatter-gather pipeline takes more than 90 minutes to get started.

Shouldn't be hard to implement, just takes some time.

**Comment by lbergelson** [comment]
Oh, interesting.  When you do a metadata query does it return `starting` as a state?

**Comment by SHuang-Broad** [comment]
yep. In the end, it took 2.5 hours to start.
Maybe such an edge case, we shouldn't care that much.

**Comment by lbergelson** [comment]
Nah, we should do it, I just have never noticed a starting state.
